package.json

```diff
@@ -17,9 +17,10 @@
   "dependencies": {
     "@dcl/catalyst-storage": "^1.0.0-20220604235145.commit-892b669",
     "@dcl/hashing": "^1.0.0",
-    "@dcl/schemas": "^5.4.2",
+    "@dcl/schemas": "tag-fast-bootstrapping-5",
```

Comment:
use fixed version before merging

Reply:
Updated the PR base branch from `main` to `feat/fetch-new-snapshots` so let's block the other one